src/index.js

```diff
@@ -16,6 +17,8 @@ checkVersion();
 
 installHooks();
 
+updateNotifier({pkg}).notify({ defer: true });
```

Comment:
Can remove `defer:true` because [it is already the default](https://github.com/yeoman/update-notifier#defer). 👍

Reply:
Ah! shoot

Reply:
Done 🙏